### DISCUSSION TITLE
Unexpected token at ''

### DISCUSSION THREAD

**Discussion by mregnauld** [codemagic-ci-cd/codemagic-docs#1403] [Q&A] [answered]
In my workflow `63eb4148cdc5d16e5878dcf1` I get the following Fastlane error:

```
809: unexpected token at ''
```

At first glance, I thought that I had a syntax error in my `Fastfile` file.
But then I tried to run that exact same Fastlane command, and it works perfectly on my local machine.

Here is my `Fastfile` file:

```Ruby
default_platform(:ios)

platform :ios do
  desc "Deploy app with screenshots to App Store Connect"
  lane :deployapp do |options|
    deliver(
      api_key_path: "./app_store_connect.json",
      app_version: options[:versionName],
      ipa: "../build/ios/ipa/mistikee.ipa",
      submit_for_review: true,
      skip_binary_upload: false,
      skip_metadata: false,
      skip_app_version_update: false,
      skip_screenshots: false,
      overwrite_screenshots: true,
      metadata_path: "./fastlane/metadata",
      screenshots_path: "./fastlane/screenshots",
      precheck_include_in_app_purchases: false,
      force: true,
      submission_information: {
        add_id_info_limits_tracking: true,
        add_id_info_serves_ads: options[:estFree],
        add_id_info_tracks_action: true,
        add_id_info_tracks_install: true,
        add_id_info_uses_idfa: true,
        content_rights_has_rights: true,
        content_rights_contains_third_party_content: true,
        export_compliance_platform: "ios",
        export_compliance_compliance_required: false,
        export_compliance_encryption_updated: false,
        export_compliance_app_type: nil,
        export_compliance_uses_encryption: false,
        export_compliance_is_exempt: false,
        export_compliance_contains_third_party_cryptography: false,
        export_compliance_contains_proprietary_cryptography: false,
        export_compliance_available_on_french_store: true
      }
    )
  end
end
```

What did I miss?

Thanks.

**Comment by mregnauld** [comment]
My bad, I forgot to set an argument for my lane.

That being said, I still have the same error here: `63eb6bb40609bfca48bbb320`, even though my arguments are correct...

**Reply by icarusdust** [reply]
Hi @mregnauld, As I am not a Fastlane expertise, I cannot give you a direct solution here. However, I can see that you had a successful build earlier with a different commit. So, I would suggest you to compare these commits to see what changes are causing this error.

**Reply by mregnauld** [reply]
Well that's my point, there wasn't any commit in between!
But yeah, I'll continue to search.
Thanks.

**Reply by icarusdust** [reply]
I see that two builds went through successfully few hours ago, so perhaps reverting to that commit could help.

**Comment by mregnauld** [comment, marked as answer]
OK so for the second error, I actually.... forgot to import an environment variable in my workflow. Silly me! :)
